fix: CI auto-merge が enablePullRequestAutoMerge 無効エラーでコケる

## 問題

CI の auto-merge ジョブが以下のエラーで失敗する:

```
GraphQL: Pull request Auto merge is not allowed for this repository (enablePullRequestAutoMerge)
Error: Process completed with exit code 1.
```

## 原因

`.github/workflows/ci.yml` の auto-merge ステップで `--auto` フラグを使用しているが、リポジトリの auto-merge 機能が無効になっているため。

## 修正方針

`--auto` フラグを削除して直接 squash merge する。
auto-merge ジョブの実行条件が満たされた時点で実行されるため `--auto` は不要。

## 修正箇所

`.github/workflows/ci.yml` の `--auto` フラグ削除。

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -224,4 +224,4 @@ jobs:
           GH_TOKEN: ${{ github.token }}
           PR_NUMBER: ${{ github.event.pull_request.number }}
         run: |
-          gh pr merge "${PR_NUMBER}" --squash --delete-branch --auto --repo ${{ github.repository }}
+          gh pr merge "${PR_NUMBER}" --squash --delete-branch --repo ${{ github.repository }}

